src/modules/flash/producer/flash_producer.cpp

```diff
@@ -555,27 +575,35 @@ spl::shared_ptr<core::frame_producer> create_producer(const core::frame_producer
 {
     auto template_host = get_template_host(dependencies.format_desc);
 
-    auto filename = env::template_folder() + L"\\" + template_host.filename;
+    auto filename = env::template_folder() + template_host.filename;
 
     if (!boost::filesystem::exists(filename))
         CASPAR_THROW_EXCEPTION(file_not_found() << msg_info(L"Could not open flash movie " + filename));
 
-    return create_destroy_proxy(spl::make_shared<flash_producer>(
-        dependencies.frame_factory, dependencies.format_desc, filename, template_host.width, template_host.height));
+    CASPAR_LOG(info) << L"Loading template host: " + url_from_path(filename);
+    return create_destroy_proxy(spl::make_shared<flash_producer>(dependencies.frame_factory,
+                                                                 dependencies.format_desc,
+                                                                 url_from_path(filename),
+                                                                 template_host.width,
+                                                                 template_host.height));
 }
 
 spl::shared_ptr<core::frame_producer> create_swf_producer(const core::frame_producer_dependencies& dependencies,
                                                           const std::vector<std::wstring>&         params)
 {
-    auto filename = env::media_folder() + L"\\" + params.at(0) + L".swf";
+    auto filename = env::media_folder() + params.at(0) + L".swf";
```

Comment:
This will break some configurations that don't end with slash

Reply:
In practice I've found this is not the case and media_folder is always returned with a slash regardless of what was in the config and so the path was always called with a double slash.

This is because trailing slash is already enforced here:
https://github.com/CasparCG/server/blob/94db7f222d852947eb61ae10b3ac6c9cbbd785be/src/common/env.cpp#L107-L110

This is used already in the image producer:
https://github.com/CasparCG/server/blob/4a8f03f3fd4eedf437e5eac3ea1ff20c817c7c22/src/modules/image/producer/image_producer.cpp#L199